# AWTI-Net
Long non-coding RNA (lncRNA) mutations are closely linked to disease, but current experimental methods are time-consuming and lack efficiency. Artificial intelligence methods are needed to accurately identify these mutations. Here, we created two high-quality lncRNA mutation datasets, LncRNAVar-Literature and LncRNAVar-GWAS, by integrating disease-associated variants from literature and GWAS studies, along with benign variants from the 1000 Genomes Project. Multidimensional characterization revealed significant differences in structural stability, miRNA targets, and species conservation between the two groups of variants, providing strong evidence for their distinction. We then proposed the AWTI-Net framework to predict the impact of lncRNA mutations. This framework integrates wavelet convolution (Conv-wave), long short-term memory (LSTM), Transformer memory (FMTrans), and an adaptive decision-making mechanism (ADMM). Extensive independent testing demonstrated AWTI-Net's exceptional predictive performance, achieving the highest AUC values compared to state-of-the-art methods (LncRNAVar-GWAS: 0.9995; LncRNAVar-Literature: 0.9949). Ablation studies highlighted the complementary strengths of Conv-wave and LSTM modules, confirming the effectiveness of the FMTrans module. Comparisons with submodels using different feature combinations showed that integrating multidimensional features significantly improved performance. Entropy-guided path selection revealed AWTI-Netâ€™s interpretable ADMM. We further demonstrated that AWTI-Net accurately identifies hepatocellular carcinoma-related lncRNA mutations, offering an efficient tool for the early diagnosis and prognosis monitoring of related diseases.
